test/angularCronJobsSpec.js

```diff
@@ -35,7 +79,7 @@ describe('AngularCronJobs', function() {
     it("cron should be set for every hour at 10 past", function() {
         var scope = $rootScope.$new();
         var view = createView(scope);
-        scope.myFrequency = {base: 2, minuteValue: [10]};
+        scope.myFrequency = {base: 2, minuteValues: [10]};
         scope.cron = cronService.setCron(scope.myFrequency);
         $rootScope.$digest();
         expect(scope.cron).toEqual('10 * * * *');
```

Comment:
I added this line to the test, and it does not fail.  Are you sure you have pulled latest when testing?